csfieldguide/config/settings/base.py

```diff
@@ -158,11 +159,13 @@
 STATIC_ROOT = str(ROOT_DIR.path("staticfiles"))
 
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#static-url
+BUILD_ROOT = os.path.join(str(ROOT_DIR.path("build")), "")
 STATIC_URL = "/staticfiles/"
 
 # See: https://docs.djangoproject.com/en/dev/ref/contrib/staticfiles/#std:setting-STATICFILES_DIRS
 STATICFILES_DIRS = [
     str(ROOT_DIR.path("static")),
+    BUILD_ROOT,
```

Comment:
If this is added, the previous line should be removed.

Reply:
I've been struggling with this bit, removing either line causes different errors, so I realise there is some overlap between them, but I can't work out why each option is causing different issues. Would be helpful if you could spin it up and have a tinker and see if you can spot what has happened to cause this.